authors/apps/articles/models.py

```diff
@@ -10,6 +10,7 @@
 
 from . import managers
 from .utils import generate_unique_slug
+from django.utils.text import slugify
```

Comment:
@Etomovich, why are you importing `slugify` in a "tags" feature?

Reply:
This is to generate a slug in Tags. My thinking is that "My Tag" and "my tag" and "my Tag" should all be a similar tag hence I have a uniform slug "my-tag" to avoid over-populating  the DB with tags